ComfortZoneApp/app/src/main/java/com/example/comfortzone/initial/data/FacebookLoginUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.example.comfortzone.initial.data;
+
+import static com.example.comfortzone.initial.LoginActivity.KEY_EMAIL;
+import static com.example.comfortzone.initial.LoginActivity.KEY_FIELDS;
+import static com.example.comfortzone.initial.LoginActivity.KEY_NAME;
+
+import android.os.Bundle;
+
+import com.facebook.AccessToken;
+import com.facebook.GraphRequest;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+import com.parse.SaveCallback;
+
+import org.json.JSONException;
+
+public class FacebookLoginUtil {
```

Comment:
Maybe we should rename it to be FacebookLoginAPI or something to indicate data fetching in the name?